sdk/communication/Azure.Communication.Rooms/tests/RoomsClientTestEnvironment.cs

```diff
@@ -1,11 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Azure.Communication.Tests;
+using Azure.Core.TestFramework;
 
 namespace Azure.Communication.Rooms.Tests
 {
     public class RoomsClientTestEnvironment : CommunicationTestEnvironment
     {
+        public const string CommunicationConnectionStringRoomsVariableName = "COMMUNICATION_CONNECTION_STRING_ROOMS";
```

Comment:
Does it mean to the INT environment, now we have added a new environment variable COMMUNICATION_CONNECTION_STRING_ROOMS with the rooms specific connection string?

Reply:
There is no differentiation for INT or Public cloud in tests. This variable is present in both INT and Public cloud key vaults. 
In INT it points to our PPE but in public it points to the same resource as the previous env variable.